articles/multifactor-authentication/guardian/user-guide.md

```diff
@@ -62,19 +62,27 @@ Depending on the type of two-factor authentication you chose when you were signi
 
 After entering your username and password, a push notification will be sent to the Guardian app on your mobile device. This notification is a login request, it includes the application name, the OS and browser of the request, the location and the date of the request. If you recognize this request as your own, tap the **Allow** button. You should now be logged in.
 
+If you do not have your mobile device available to you during sign in, you can enter the recovery code that was given when you signed up by clicking the "enter the code" link.
```

Comment:
The "enter the code" link is for entering the OTP code, not the recovery code. And you need your device, but this helps if you don't have internet or there's a problem with the push notification service.
This is mixing top code vs recovery code